packages/polyfill-library/polyfills/Array/of/polyfill.js

```diff
@@ -0,0 +1,53 @@
+/*! https://mths.be/array-of v0.1.0 by @mathias */
```

Comment:
This file can be deleted as it is automatically built from the `array.of` npm package. I'm not too sure how it ended up being commited as it is included in the [.gitignore](https://github.com/Financial-Times/polyfill-service/pull/1267/files#diff-a084b794bc0759e7a6b77810e01874f2R11) file.